src/documents/ConditionalSubpanels.md

```diff
@@ -17,7 +17,7 @@ The relationship between the controlling boolean preference and its dependent su
 }
 ```
 
-**Notes:**
+## Notes:
```

Comment:
I think we can remove the colon if we are now marking up as a heading -- "Notes" rather than "Notes:".

Reply:
I see that in other places in the docs, we have many headings that end in a colon. I'm fine if we leave this heading as is. You can ignore this one.